jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -286,6 +304,10 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_PROMISE_UL, "Promise")
 #if !defined (CONFIG_DISABLE_MATH_BUILTIN)
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SQRT1_2_U, "SQRT1_2")
 #endif
+#if !defined (CONFIG_DISABLE_ES2015_SYMBOL_BUILTIN)
+LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SYMBOL_LEFT_PAREN_UL, "Symbol(")
+LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SYMBOL_DOT_UL, "Symbol.")
```

Comment:
The reason behind the 2 extra magic strings that these string should be created run-time anyway. It reduces the number of memory allocations, also if for some reason a `Symbol.` string would be constructed somehow during the execution it does not requires to allocate memory for it, since it is a magic string. But in the other hand, yes it consumes + 14B memory, but I think it's an acceptable deal.